contribute/backend/style-guide.md

```diff
@@ -66,65 +66,62 @@ Use [`t.Cleanup`](https://golang.org/pkg/testing/#T.Cleanup) to clean up resourc
 
 ### Mock
 
-Optionally, we use [`mock.Mock`](https://github.com/stretchr/testify#mock-package) package to generate mocks. This is
+Optionally, we use [`mock.Mock`](https://github.com/stretchr/testify#mock-package) package to write mocks. This is
```

Comment:
You can use to autogen the mocks: https://vektra.github.io/mockery/latest/

Reply:
There's a section for mockery below this one